examples/temporal_graph_learning/gnn_encoder.py

```diff
@@ -0,0 +1,145 @@
+from graphstorm import model as gsmodel
+
+from model_utils import get_unique_nfields
+from model_utils import to_per_field_nfeats
+from model_utils import rel_field_map
+from model_utils import rel_name_map
+from model_utils import average_over_fields
+from model_utils import get_unique_etype_triplet
+from model_utils import get_temporal_restrict_fields
+
+from model_utils import merge_multi_blocks, get_merge_canonical_etype_mapping
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from dgl.nn import GATConv
+
+class RelationalGraphEncoder(gsmodel.gnn_encoder_base.GraphConvEncoder):
```

Comment:
where do you put temporal encoding?

Reply:
we need to have one place to explain TGAT and how TGAT is implemented in this code base.

Reply:
I added a paragraph in `README.md` to explain how we utilize temporal information based on the GraphStorm's  RGAT.